src/index.html

```diff
@@ -7,8 +7,91 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        class="moyo_logo"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+            APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+            SAMSUNG
+            </a>
```

Comment:
```suggestion
            >
              SAMSUNG
            </a>
```
the same is with all the links 😉